src/schedule-and-details/utils.js

```diff
@@ -0,0 +1,76 @@
+import moment from 'moment';
+
+import { CERTIFICATE_DISPLAY_BEHAVIOR } from './schedule-section/certificate-display-row';
+import messages from './messages';
+
+const DATE_TIME_FORMAT = 'YYYY-MM-DDTHH:mm:ss\\Z';
+
+const convertToDateFromString = (dateStr) => {
+  if (!dateStr) {
+    return '';
+  }
+
+  return moment(dateStr).utc().toDate();
+};
+
+const isDateBefore = (dateFormer, dateLatter, checkExists = true) => {
+  if (checkExists && (!dateFormer || !dateLatter)) {
+    return false;
+  }
+  return new Date(dateFormer) <= new Date(dateLatter);
```

Comment:
Please add a space as in the functions above and below, for uniformity :)

Reply:
fixed